multichain-testing/Makefile

```
@@ -79,6 +79,11 @@ override-chain-registry:
 	node_modules/.bin/tsx scripts/fetch-starship-chain-info.ts && \
 	node_modules/.bin/tsx scripts/deploy-cli.ts src/revise-chain-info.builder.js
 
+register-bank-assets:
+	node_modules/.bin/tsx scripts/fetch-starship-chain-info.ts && \
+	node_modules/.bin/tsx scripts/deploy-cli.ts src/register-interchain-bank-assets.builder.js \
+		assets="$$(node_modules/.bin/tsx scripts/make-bank-asset-info.ts)"
+
```

what does it cost to get the scripts to know how to execute themselves?

```suggestion
register-bank-assets:
	./scripts/fetch-starship-chain-info.ts && \
	./scripts/deploy-cli.ts src/register-interchain-bank-assets.builder.js \
		assets="$$(./scripts/make-bank-asset-info.ts)"

```

---

I'll see where we land in #10605 and incorporate that approach

---

yeah; don't worry about it for this PR